app/views/content/train-to-be-a-teacher/_nav-cards.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="container">
+    <div class="category__nav-cards">
+        <a href="/train-to-be-a-teacher/if-you-have-a-degree" class="category__nav-card">
+            <h2>If you have a degree</h2>
```

Comment:
@valentine-carter are we missing a card on 'If you don't have a degree'?

It is important that we signpost people to undergraduate courses (as we do from Ways to train currently)?

Reply:
I've addded a card and new page. I'm not sure about having it at this level but at this point, given the time, I think we should just add it and then revisit it later if we can.

Reply:
This might fit into the new content requested by the pipeline team in future

Reply:
Promo module on this 'If you don't have a degree' page could be TTAs - fits in with early engagement advisers potentially?